app/controllers/participants_controller.rb

```diff
@@ -456,7 +456,7 @@ def respond_to_generate_xlsx
 		
 		if request.xhr?
 			headers["Content-Type"] = "text/javascript"
-			return render(:template => "participants/check_export_status.js.rjs", :format => 'js')
+			return render(:template => "participants/check_export_status.js.erb", :format => 'js')
```

Comment:
Line is too long. [89/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.